Form-Controls/styles.css

```diff
@@ -0,0 +1,89 @@
+/* General styles */
+@import url("https://fonts.googleapis.com/css2?family=Jockey+One&family=Manrope:wght@200;300;400;500;600&family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --background: #f7fafc;
+  --brand: #1a3647;
+  --font: "Jockey One";
+}
+
+body {
+  display: grid;
+  height: 100vh;
+  max-height: 100vh;
```

Comment:
Rarely set a max height. Remember all pro layouts must work at 200% text size.

In fact , first make a prediction, and then zoom your layout to 200% and see what happens to the max height. Was it what you expected? Now set the root font size to 32px.  What happens?